hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -475,10 +476,15 @@ simulatedChainAndNetwork = do
     ConnectToChain
       { chainComponent = \node -> do
           atomically $ modifyTVar nodes (node :)
+          peersVar <- newTVarIO mempty
```

Comment:
We should reuse the `nodes` for keeping the peers .. if we want/need to mock that here at all.

Reply:
as modifyPeers is not expected to be used here, will going to just remove the implementation for it on this spec.